src/components/layouts/MainPage.js

```diff
@@ -77,6 +77,7 @@ class MainPage extends React.Component {
                         <Row>{confData.footerText}</Row>
                         <Row>{"Deploy date: " + confData.deployDate + " from branch: "
                             + confData.deployBranch + " and commit number: " + confData.deployCommit}</Row>
+                        <Row><a href="report.html" target="_blank">{"See the bundle.js analyze"}</a></Row>
```

Comment:
Is it possible to display such stats only on deployment to https://libamtrack.github.io/web_dev ?
For regular users it may be a bit misterious ?

Reply:
done